selftests/run_coverage

```diff
@@ -6,7 +6,9 @@
 # Author: Lukas Doktor <ldoktor@redhat.com>
 
 coverage erase
-coverage run --include "avocado/*" ./selftests/run
+coverage rm .coverage.*
```

Comment:
Oh, sorry, obviously it should be just `rm .coverage.*` to physically remove the files... Anyway I'll send a rebased v2 with this line fixed.